src/ontology/Makefile

```diff
@@ -1,4 +1,4 @@
-all: hp.obo hp.owl all-subsets hp-inferred.obo
+all: hp.obo hp.owl hp-inferred.obo
```

Comment:
@cmungall please comment if this change has consequences that we are maybe not aware of. thanks